frontend/device/pocketbook/device.lua

```diff
@@ -206,156 +206,207 @@ function PocketBook:getDeviceModel()
--- PocketBook Lux 4
-local PocketBook627 = PocketBook:new{
-    model = "PBLux4",
+-- PocketBook Basic Lux 2 (616)
+local PocketBook616 = PocketBook:new{
+    model = "PBBLux2",
     display_dpi = 212,
+    isTouchDevice = no,
+    hasDPad = yes,
+    hasFewKeys = yes,
 }
 
--- PocketBook Touch HD
-local PocketBook631 = PocketBook:new{
-    model = "PBTouchHD",
-    display_dpi = 300,
-}
-
--- PocketBook Touch HD Plus
-local PocketBook632 = PocketBook:new{
-    model = "PBTouchHDPlus",
-    display_dpi = 300,
-    isAlwaysPortrait = yes,
+-- PocketBook Touch (622)
+local PocketBook622 = PocketBook:new{
+    model = "PBTouch",
+    display_dpi = 167,
+    hasFrontlight = no,
 }
 
--- PocketBook Lux 3
-local PocketBook626 = PocketBook:new{
-    model = "PBLux3",
+-- PocketBook Touch Lux (623)
+local PocketBook623 = PocketBook:new{
+    model = "PBTouchLux",
     display_dpi = 212,
 }
 
--- PocketBook Basic Touch
+-- PocketBook Basic Touch (624)
 local PocketBook624 = PocketBook:new{
     model = "PBBasicTouch",
+    display_dpi = 167,
     hasFrontlight = no,
-    display_dpi = 166,
 }
 
--- PocketBook Basic Touch 2
+-- PocketBook Basic Touch 2 (625)
 local PocketBook625 = PocketBook:new{
     model = "PBBasicTouch2",
+    display_dpi = 167,
     hasFrontlight = no,
-    display_dpi = 166,
 }
 
--- PocketBook Touch
-local PocketBook622 = PocketBook:new{
-    model = "PBTouch",
-    hasFrontlight = no,
-    display_dpi = 166,
+-- PocketBook Touch Lux 2 / Touch Lux 3 (626)
+local PocketBook626 = PocketBook:new{
+    model = "PBLux3",
+    display_dpi = 212,
 }
 
--- PocketBook Touch Lux
-local PocketBook623 = PocketBook:new{
-    model = "PBTouchLux",
+-- PocketBook Touch Lux 4 (627)
+local PocketBook627 = PocketBook:new{
+    model = "PBLux4",
     display_dpi = 212,
 }
 
--- PocketBook InkPad 3
-local PocketBook740 = PocketBook:new{
-    model = "PBInkPad3",
+-- PocketBook Touch Lux 5 (628)
+local PocketBook628 = PocketBook:new{
+    model = "PBTouchLux5",
+    display_dpi = 212,
     isAlwaysPortrait = yes,
-    display_dpi = 300,
 }
 
--- PocketBook InkPad 3 Pro
-local PocketBook740_2 = PocketBook:new{
-    model = "PBInkPad3Pro",
-    isAlwaysPortrait = yes,
+-- PocketBook Sense / Sense 2 (630)
+local PocketBook630 = PocketBook:new{
+    model = "PBSense",
+    display_dpi = 212,
+}
+
+-- PocketBook Touch HD / Touch HD 2 (631)
+local PocketBook631 = PocketBook:new{
+    model = "PBTouchHD",
     display_dpi = 300,
 }
 
--- PocketBook InkPad X
-local PocketBook1040 = PocketBook:new{
-    model = "PB1040",
+-- PocketBook Touch HD Plus / Touch HD 3 (632)
+local PocketBook632 = PocketBook:new{
+    model = "PBTouchHDPlus",
+    display_dpi = 300,
     isAlwaysPortrait = yes,
-    display_dpi = 227,
 }
 
--- PocketBook Sense
-local PocketBook630 = PocketBook:new{
-    model = "PBSense",
-    display_dpi = 212,
+-- PocketBook Color (633)
+local PocketBook633 = PocketBook:new{
+    model = "PBColor",
+    display_dpi = 300,
+    hasColorScreen = yes,
```

Comment:
Doesn't need:

```
    has3BytesWideFrameBuffer = yes,
    canUseCBB = no, -- 24bpp
```

¿?

Reply:
I mean: I'm not sure what I'm talking about, but the last line of the logs in https://github.com/koreader/koreader/issues/6479#issue-674324708, before the segmentation fault, is `FB: bpp          : 24`

pinging @NiLuJe (but he's on holidays, so I would say add the definition anyways and revert back later if I was utterly wrong, which is probable)

Reply:
I mean: I'm not sure what I'm talking about, but the last line of the logs in https://github.com/koreader/koreader/issues/6479#issue-674324708, before the segmentation fault, is `FB: bpp          : 24`

pinging @NiLuJe (but he's on holidays, so I would say add the definition anyways and revert back later if I was utterly wrong, which is probable)

Reply:
It'll need someone with the device and all, but https://github.com/koreader/koreader-base/pull/732 with its `has3BytesWideFrameBuffer = yes,` might potentially be relevant.

Reply:
See <https://goodereader.com/blog/electronic-readers/review-of-the-pocketbook-color-lux-ereader>.

Reply:
The Color 633 is different though - https://pocketbook.ch/en-ch/catalog/color/color-ch, hence my uncertainty, but #6479's creator might be able to test it out?

Reply:
yep, that's the point. merge it but don't close #6479 yet. Ping OP when a nightly is available and see if that automagically fixes it or at least something in the logs changed :p

Reply:
Don't quote me on this, but I faintly remember reading somewhere today, that the 633 has 300dpi for black & white/grayscale and 100dpi for color or something of the sort. No idea whether that's important or useful information :/

Reply:
I didn't say to add that line, just that it's something that's there if necessary. But in the heat I overlooked that @pazos already said that.

Reply:
> No idea whether that's important or useful information :/

Yes and no; I doubt it has anything to do with what we have to feed it. ;-)